tsdfileapi/api.py

```diff
@@ -293,8 +293,8 @@ def enforce_group_logic(self, group_name, group_memberships, tenant, group_confi
             if group_config['enforce_membership']:
                 assert group_name in group_memberships
         except (AssertionError, Exception) as e:
-           logging.error('user not member of group')
-           raise e
+            logging.error(f'user {tenant} not member of group {group_name}')
```

Comment:
In this case the tenant is the project, not the user

Reply:
ok.